website/source/docs/commands/config/list.html.md.erb

```diff
@@ -11,8 +11,9 @@ Command: `consul config list`
 The `config list` command lists all given config entries of the given kind.
 See the
 [agent configuration](/docs/agent/options.html#enable_central_service_config)
```

Comment:
I don't think we should put this here - in the future these commands will manage other config entries that are nothing to do with service config and not affected by that agent setting. Maybe just keep the new sentence that is more generic and doesn't mention "service"?